apps/shopify/src/collectionPagination.js

```diff
@@ -8,12 +9,17 @@ const makePagination = async (sdk) => {
     fetchProducts: async function (search, PER_PAGE) {
       const query = { query: search };
 
-      return await this.shopifyClient.collection.fetchQuery({
+      const response = await this.shopifyClient.collection.fetchQuery({
```

Comment:
Same here. It's better to replace abstact 'response' with something more literal.